samples/contrib/e2e-mnist/mnist-pipeline.ipynb

```diff
@@ -1,19 +1,43 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MNIST End to End examples with Kubeflow compoenents\n",
+    "\n",
+    "This pipeline contains 5 steps, it finds the best hyperparameter using Katib, creates PVC for storing models, processes the hyperparameter results, distributedly trains the model on TFJob with the best hyperparameter using more iterations, and finally serves the model using KFServing. You can visit this [medium blog](https://medium.com/@liuhgxa/an-end-to-end-use-case-by-kubeflow-b2f72b0b587) for more details on this pipeline."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define the model name and Kubeflow user namespace\n",
+    "\n",
+    "Change the below cell with a name you want to use for this pipeline and which namespace you want to execute on Kubeflow."
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": null,
```

Comment:
what does null signify here?

Reply:
execution_count shows the order of the executed cells. Null here means that we cleared the execution results.